### PR TITLE
SB-970: DownloadCount attribute for Nuget RSS feed

### PR DESCRIPTION
**Pull request by sbespalov**

### CODE REVIEW

File: strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryPath.java [hunk 1]
```diff
@@ -44,8 +46,12 @@ public Path getTarget()
     {
         return target;
     }
+    
+    public ArtifactEntry getArtifactEntry() {
```
Comment (carlspring) [inline strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryPath.java]: Cosmetic: The indentation of this method seems incorrect.

File: strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryPath.java [hunk 2]
```diff
@@ -44,8 +46,12 @@ public Path getTarget()
...
+		return artifactEntry;
+	}
 
-    public RepositoryFileSystem getFileSystem()
+	public RepositoryFileSystem getFileSystem()
```
Comment (carlspring) [inline strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryPath.java]: Cosmetic: The indentation of this method seems incorrect.

File: strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/nuget/NugetPackageController.java [hunk 3]
```diff
@@ -154,19 +179,100 @@ public ResponseEntity deletePackage(@RequestHeader(name = "X-NuGet-ApiKey", requ
...
+
+            @Override
+            protected String getRootUri()
+            {
+                return feedId;
+            }
+        };
+    }
+
+    protected Comparator<PackageEntry> getPackageComparator(final String orderByClause) {
```
Comment (carlspring) [inline strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/nuget/NugetPackageController.java]: Cosmetic: The indentation of this method seems incorrect.

File: strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryPath.java (hunk 1 above)
Comment (sbespalov) [reply strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryPath.java]: fixed
sorry :)

File: strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryPath.java (hunk 2 above)
Comment (sbespalov) [reply strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryPath.java]: fixed

File: strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/nuget/NugetPackageController.java (hunk 3 above)
Comment (sbespalov) [reply strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/nuget/NugetPackageController.java]: fixed

File: strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/nuget/NugetPackageControllerTest.java [hunk 4]
```diff
@@ -395,4 +397,23 @@ public void testRemoteProxyGroup()
...
+               .get(getContextBaseUrl() + "/storages/public/nuget-public/FindPackagesById()?id=NHibernate&$orderby=Version")
+               .then()
+               .statusCode(HttpStatus.OK.value())
+               .and()
+               .assertThat()
+               .body("feed.title", equalTo("Packages"))
+               .and()
+               .assertThat()
+               .body("feed.entry[0].title", equalTo("NHibernate"))
+               .body(hasXPath("/atom:feed/atom:entry[last()]/m:properties/d:IsLatestVersion[text()='true']"));
```
Comment (sbespalov) [inline strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/nuget/NugetPackageControllerTest.java]: @carlspring @fuss86 @steve-todorov 
guys, do anyone knows XPAth?
Need help to make this expression work :)

**Comment by carlspring** [comment]
Thanks!

**Pull request merged**